src/Val.cc

```diff
@@ -2762,6 +2846,24 @@ void TableVal::RebuildTable(ParseTimeTableState ptts) {
         Assign(std::move(key), std::move(val));
 }
 
+size_t TableVal::Hash() const {
```

Comment:
Just wondering, so this would allow to put `TableVal` into a `std::set` for example?

But if you get a reference to a specific entry and modify it by adding/removing an entry, the hash would change without the `std::set` noticing or putting it in a different bucket. Isn't this dangerous?